### PR TITLE
More varioplot improvements

### PR DESCRIPTION
**Pull request by Lenart12**
Some new stuff for varioplot :^)
* Wind indicator in varioplot
* Unit for wind speed can be changed
* Current circling info in varioplot (duration + gain)
* Small changes/cleanunp with wind step processing
image

Also: I updated readme to add images and links to manual, and rewrote the manual to use markdown. [Check my fork how it looks](https://github.com/lenart12/myvario)

### CODE REVIEW

File: source/MyProcessing.mc [hunk 1]
```diff
@@ -606,44 +606,35 @@ class MyProcessing {
...
     if(LangUtils.notNaN(self.fHeading) && LangUtils.notNaN(self.fGroundSpeed) && self.fHeading != null && self.fGroundSpeed != null) {
-      self.iAngle = ((self.fHeading * 57.2957795131f).toNumber()) % 360;
+      self.iAngle = Math.toDegrees(self.fHeading).toNumber() % 360;
       self.fSpeed = self.fGroundSpeed;      
     } else {
       return;
     }
 
-    self.iWindSector = (self.iAngle + (360 / self.DIRECTION_NUM_OF_SECTORS / 2)) % 360 / (360 / self.DIRECTION_NUM_OF_SECTORS);
+    self.iWindSector = (self.iAngle + (180 / self.DIRECTION_NUM_OF_SECTORS)) % 360 / (360 / self.DIRECTION_NUM_OF_SECTORS);
```
Comment (Lenart12) [inline source/MyProcessing.mc:618]: 360 / 2 changed to 180

File: source/MyProcessing.mc [hunk 2]
```diff
@@ -606,44 +606,35 @@ class MyProcessing {
...
-      else {
-        if(self.iWindOldSector == self.iWindSector) {
-          //Same sector
-          self.bNotCirclingCount += 1;
-        }
-        else {
-          //More than 360/num of sectors, discard data
-          self.iWindSectorCount = 0;
-        }
-      }
```
Comment (Lenart12) [inline source/MyProcessing.mc:646]: This block of IF statements is functionaly the same, just neater :^)

File: source/MyProcessing.mc [hunk 3]
```diff
@@ -355,6 +352,9 @@ class MyProcessing {
       if(self.iAccuracy > Pos.QUALITY_LAST_KNOWN) {
         self.iPositionEpoch = _iEpoch;
 
+        // ... wind
+        self.windStep();
```
Comment (Lenart12) [inline source/MyProcessing.mc:359]: My reasoning for moving this is I think we should always use stateful position for wind? LMK if this is wrong

Comment (ydutertre) [reply source/MyProcessing.mc:359]: Yeah I think that's fine!

File: source/MyProcessing.mc [hunk 4]
```diff
@@ -671,4 +671,16 @@ class MyProcessing {
     }
   }
 
+  function getWindColor(_fSpeed as Float) as Number {
```
Comment (ydutertre) [inline source/MyProcessing.mc:674]: this is so cool :)

**Comment by ydutertre** [comment]
Merging, thank you! Will publish on app store shortly

**Pull request merged**